Humble CI is broken

[Humble CI is failing](https://github.com/ros-planning/moveit2/actions/runs/3744195554) with the following error:
```
--- stderr: moveit_ros_planning_interface
/home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp:54:10: fatal error: moveit_msgs/srv/execute_known_trajectory.hpp: No such file or directory
   54 | #include <moveit_msgs/srv/execute_known_trajectory.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```
I don't understand why. The linked GHA build [succeeded on Dec 20](https://github.com/ros-planning/moveit2/actions/runs/3744195554/attempts/1), but [fails now](https://github.com/ros-planning/moveit2/actions/runs/3744195554/attempts/2) (I just retriggered the build).
Hence, it is not related to the source itself. It must be related to GHA, industrial_ci, or the docker image. 
However, a local build using industrial_ci and the humble docker image succeeded for me. I' don't know what I'm missing...

--- a/moveit2.repos
+++ b/moveit2.repos
@@ -3,12 +3,12 @@ repositories:
   moveit_msgs:
     type: git
     url: https://github.com/ros-planning/moveit_msgs.git
-    version: ros2
+    version: humble
   # https://github.com/ros-planning/moveit_resources/pull/147
   moveit_resources:
     type: git
     url: https://github.com/ros-planning/moveit_resources.git
-    version: ros2
+    version: humble
   ros2_control:
     type: git
     url: https://github.com/ros-controls/ros2_control.git

